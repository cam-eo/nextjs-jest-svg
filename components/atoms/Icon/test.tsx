import { Icon } from ".";
import { render } from "@testing-library/react";

describe("Icon", () => {
  it("should render Icon", () => {
    const { asFragment } = render(<Icon name="chevron-left" />);

    expect(asFragment()).toMatchSnapshot();
  });
});
