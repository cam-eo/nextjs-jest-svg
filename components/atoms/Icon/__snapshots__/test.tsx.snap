// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render Icon 1`] = `
<DocumentFragment>
  <svg
    fill="#123456"
    height="24"
    width="24"
  />
</DocumentFragment>
`;
